tests/integration/golang/mlflow/run/create_test.go

```diff
@@ -248,26 +251,26 @@ func (s *CreateRunTestSuite) Test_Error() {
 		{
 			name: "CreateRunWithNotExistingExperiment",
 			request: request.CreateRunRequest{
-				ExperimentID: fmt.Sprintf("%d", nonExistingExperimentID),
+				ExperimentID: fmt.Sprintf("%d", -1),
```

Comment:
why switch from descriptive variable to a magic number?

Reply:
but probably name -> `CreateRunWithNotExistingExperiment` already says what we try to do and construction -> `ExperimentID: fmt.Sprintf("%d", -1)` immediately gives you context that this is experiment id.

Reply:
what is really confusing is -> `nonExistingExperimentID := *experiment.ID + 1`. so I just changed this moment

Reply:
ok